Pokedex/Pokedex/PokemonTableViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  PokemonTableViewController.swift
+//  Pokedex
+//
+//  Created by Gladymir Philippe on 7/17/20.
+//  Copyright © 2020 Gladymir Philippe. All rights reserved.
+//
+
+import UIKit
+
+class PokemonTableViewController: UITableViewController {
+    
+    var pokemonController = PokemonController()
+    
+    var pokemon: Pokemon! {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.reloadData()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableView.reloadData()
+    }
+
+    // MARK: - Table view data source
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return pokemonController.savedPokemon.count ?? 0
```

Comment:
```suggestion
        return pokemonController.savedPokemon.count
```
Since the pokemonController is being instantiated on this view controller the right side of your default will never run.